### PR TITLE
Update ANT info

### PR DESCRIPTION
**Pull request by bpierre**

### CODE REVIEW

File: tokens/eth/0x960b236A07cf122663c4303350609A66A7B288C0.json [hunk 1]
```diff
@@ -4,11 +4,11 @@
   "decimals": 18,
   "name": "Aragon",
   "ens_address": "",
-  "website": "https://aragon.org",
+  "website": "https://aragon.org/",
```
Comment (ligi) [inline tokens/eth/0x960b236A07cf122663c4303350609A66A7B288C0.json]: what advantage do you see in adding this?

Comment (bpierre) [reply tokens/eth/0x960b236A07cf122663c4303350609A66A7B288C0.json]: It doesn’t change anything really, I like to keep the `/` explicit in URLs but it’s a personal preference. Happy to revert this change, let me know :-)

Comment (ligi) [reply tokens/eth/0x960b236A07cf122663c4303350609A66A7B288C0.json]: yea - please remove it - my personal preference is to remove it as I see only disadvantages (consumes one more byte and IMHO looks ugly) and no advantages.

Comment (bpierre) [reply tokens/eth/0x960b236A07cf122663c4303350609A66A7B288C0.json]: Done!

**Comment by ligi** [comment]
Thanks - will merge now - just a technical problem - the empty merge commit at the top (https://github.com/ethereum-lists/tokens/pull/302/commits/ea24160d29055450eac9248e8e83f67fac8a0f55) breaks my CI pipeline and so github blocks the merge - either you remove (useless anyway)  or I need to find a way to work around it

**Pull request merged**